api/helm/api/values.yaml

```diff
@@ -8,27 +8,28 @@ secret: ChangeMe
 corsAllowOrigin: http://example.com
```

Comment:
Change for "^https?://[a-z\]*\.api-platform.com$'" ?

Reply:
Change for `*`, we want users to be able to try the API with theirs clients.